common/prettify.pm

```diff
@@ -587,6 +587,8 @@ use warnings;
 
 use FileHandle;
 
+use URI::URL;
```

Comment:
URI::URL isn't part of the Perl core modules, so I don't think we should be using it. Even some of the core modules give us headaches on some machines, this one is guaranteed to cause us trouble.

Reply:
removed